components/centraldashboard/public/components/registration-page.js

```diff
@@ -84,7 +85,7 @@ export class RegistrationPage extends utilitiesMixin(PolymerElement) {
             el.style.width = `${el.scrollWidth}px`;
         });
         this.sleep(2000).then(async () => {
-            this.dispatchEvent(new CustomEvent('flowcomplete'));
+            this.fire('flowcomplete');
```

Comment:
It looks like `fire` was maintained for backwards compatibility with older versions of Polymer, but you override it with a custom definition. https://polymer-library.polymer-project.org/3.0/docs/devguide/events#custom-events.

Should the name differ just to avoid any confusion?

Reply:
`.fire` -> `.fireEvent`